lib/postzord/dispatcher.rb

```diff
@@ -170,14 +170,7 @@ def notify_local_users(local_people)
   def notify_users(users)
     return unless users.present? && @object.respond_to?(:persisted?)
 
-    #temp hax
-    unless object_is_related_to_diaspora_hq?
-      Workers::NotifyLocalUsers.perform_async(users.map{|u| u.id}, @object.class.to_s, @object.id, @object.author.id)
-    end
-  end
-
-  def object_is_related_to_diaspora_hq?
-    (@object.author.diaspora_handle == 'diasporahq@joindiaspora.com' || (@object.respond_to?(:relayable?) && @object.parent.author.diaspora_handle == 'diasporahq@joindiaspora.com'))
+    Workers::NotifyLocalUsers.perform_async(users.map{|u| u.id}, @object.class.to_s, @object.id, @object.author.id)
```

Comment:
Space missing to the left of {.<br>Space missing inside }.<br>Pass `&:id` as an argument to `map` instead of a block.